frontend/src/app/libs/user-profile-page/user-profile-page.component.ts

```diff
@@ -0,0 +1,74 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { Subject } from 'rxjs';
+import { takeUntil } from 'rxjs/operators';
+import { FormBuilder, FormGroup } from '@angular/forms';
+import { Router } from '@angular/router';
+import { EntityWrapper } from '../auth/models/entity-wraper';
+import { EntityStatus } from '../auth/models/entity-status';
+import { User } from '../auth/models/user';
+import { AuthFacadeService } from '../auth/store/auth/auth.facade';
+
+@Component({
+	selector: 'app-user-config',
+	templateUrl: './user-profile-page.component.html',
+	styleUrls: ['./user-profile-page.component.scss'],
+})
+export class UserProfilePageComponent implements OnInit, OnDestroy {
+	private destroy$: Subject<boolean> = new Subject<boolean>();
+	public profileForm: FormGroup;
+	public user: User;
+
+	constructor(private authFacadeService: AuthFacadeService, private router: Router, private formBuilder: FormBuilder) {
+		this.profileForm = formBuilder.group({
+			email: [''],
+			password: [''],
+			mobilePhone: [''],
+			firstName: [''],
+			secondName: [''],
+		});
+	}
+
+	private setValuesToForm(): void {
+		this.profileForm.setValue({
+			email: this.user.email,
+			password: this.user.password,
+			firstName: this.user.firstName,
+			secondName: this.user.secondName,
+			mobilePhone: this.user.mobilePhone,
+		});
+	}
+
+	private getChangesFromForm(): User {
+		const valuesFromForm: User = {
+			id: this.user.id,
+			isAdmin: false,
```

Comment:
lets avoid this line, i mean here you can just store and modify `User` instance inside this form, and pass updated version of current user to `.updateUser()`